### PR TITLE
fix style on create community first step

### PR DESCRIPTION
**Pull request by germanurrus**
- Small pr to fix style on mobile create community flow: 
Before:
Screen Shot 2022-06-10 at 11 08 25
After:
Screen Shot 2022-06-10 at 11 06 52

Also fix for stepper, line separation was at the bottom:
Before:
Screen Shot 2022-06-10 at 11 10 08
After:
Screen Shot 2022-06-10 at 12 48 07

### CODE REVIEW

File: frontend/packages/client/src/components/Svg.js [hunk 1]
```diff
@@ -617,10 +617,10 @@ export const ValidCheckMark = () => (
...
       d="M10 17.5C14.1421 17.5 17.5 14.1421 17.5 10C17.5 5.85786 14.1421 2.5 10 2.5C5.85786 2.5 2.5 5.85786 2.5 10C2.5 14.1421 5.85786 17.5 10 17.5Z"
       stroke="#5ECE84"
-      stroke-width="2"
-      stroke-miterlimit="10"
-      stroke-linecap="round"
-      stroke-linejoin="round"
+      strokeWidth="2"
+      strokeMiterlimit="10"
+      strokeLinecap="round"
+      strokeLinejoin="round"
```
Comment (germanurrus) [inline frontend/packages/client/src/components/Svg.js:623]: change to camel case

File: frontend/packages/client/src/components/StepByStep/index.js [hunk 2]
```diff
@@ -91,7 +91,7 @@ function StepByStep({
       );
     }
     if (stepLabel) {
-      stepClasses.push("mb-6");
+      stepClasses.push("mb-6 is-align-items-center");
```
Comment (germanurrus) [inline frontend/packages/client/src/components/StepByStep/index.js:94]: only align if label is present

**Comment by germanurrus** [comment]
> @germanurrus will this also center align the text for the step numbers? if not could you update PR to resolve this as well Screen Shot 2022-06-13 at 9 46 10 AM

yes, the outer container is aligned center and the div that contains the number is also align centered: 
Screen Shot 2022-06-13 at 14 43 38
and then the inner div that holds the number also is align center:
Screen Shot 2022-06-13 at 14 44 00

**Review by dbslone — APPROVED** [review]

**Comment by sarahYounes-DapperLabs** [comment]
reviewed

**Pull request merged**